bin/ember

```diff
@@ -27,5 +27,8 @@ resolve('ember-cli', {
     cliArgs: process.argv.slice(2),
     inputStream: process.stdin,
     outputStream: process.stdout
-  }).then(exit);
+  }).then(function(result) {
+    var exitCode = typeof result === 'object' ? result.exitCode : result;
```

Comment:
This file's change should be perfectly backwards compatible (we allow for `result` to either be an object (new thing) or a number (old thing)).  We should add a comment saying why we are doing the ternary (and what version it was added in so we know when we can remove it).

@stefanpenner - In theory we could avoid changing this file, but then we could not test the UI output in acceptance tests...